bool is_rgb
---
sensor_msgs/Image image
sensor_msgs/PointCloud2 pointcloud
